ROMFS/px4fmu_common/init.d/airframes/2106_albatross

```diff
@@ -35,11 +31,60 @@ param set-default FW_WR_IMAX 0.8
 param set-default FW_WR_P 1
 param set-default FW_W_RMAX 0
 
-# set disarmed value for the ESC
-param set-default PWM_MAIN_DISARM 1000
+param set-default CA_AIRFRAME 1
+param set-default CA_ROTOR_COUNT 1
+param set-default CA_ROTOR0_PX 0.3
```

Comment:
I would not set this - has no effect anyway but could be confusing. And anyway, the albatross would have a pusher and not a puller.

Reply:
Alright, removed